tests/test_sensor_snooze.py

```
@@ -0,0 +1,224 @@
+
+class TestIsActiveNotification:
+    """Unit tests for the _is_active_notification logic in _process_raw_notifications.
+
+    Because _is_active_notification is a nested closure, its logic is replicated
+    here using the public _coerce_datetime method so the behaviour is tested
+    without requiring a full HA environment setup.
+    """
+
+    NOW = datetime.datetime(2024, 6, 1, 8, 0, 0, tzinfo=UTC)
+
+    `@staticmethod`
```

Pylint (4.0.5)

[error] Parsing failed: 'invalid syntax (test_sensor_snooze, line 161)'

(E0001)